classes/PipelineRequest/mock.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+/**
+ * Mocked PipelineRequest.
+ * Use it any time you need to mock a PipelineQuest.
+ *
+ * Usage:
```

Comment:
Usage is not needed. We have a `README.md`. Please remove from the code.

Reply:
done